### PR TITLE
issue-11 Create ad and content layouts

### LINKED ISSUE #11: [Layout] - Fixed Page layout  for ads and content placement
Fixed page layout for  placement of ads and other content pieces across the site. Would be good if we can template out the layout so it can be reused globally.

- [ ] Mobile version
- [ ] Desktop version
- [ ] Page Scroll behavior (becomes fixed after scrolling half way through the page)

As we have some pieces that can be changed by the authors, We can also try to make the template in a fragment form such that authors can edit the elements in the layout if they want to.

**Note** : similar was done with another crn site. We can take inspiration from 
- https://www.theiotintegrator.com/
- Source at https://github.com/hlxsites/channelco-iot

IMG_2040

### PR DESCRIPTION
**Pull request by samuelopra**
Fix #11 

Test URLs:
- Before: https://main--channelco-crn-com--hlxsites.hlx.page/
- After: https://issue-11-layout--channelco-crn-com--hlxsites.hlx.page/

https://issue-11-layout--channelco-crn-com--hlxsites.hlx.page/news/internet-of-things/5-industrial-iot-solutions-that-are-solving-big-problems

### CODE REVIEW

File: scripts/delayed.js [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+async function loadAdFragment() {
+  const adFragmentContainer = document.getElementById('right-ad-fragment-container');
+  if (adFragmentContainer) {
+    const path = '/fragments/global-layout-right-fragment'; // Updated the fragment path
+    const resp = await fetch(`${path}.plain.html`);
+    if (resp.ok) {
+      const fragmentHTML = await resp.text();
+      adFragmentContainer.innerHTML = fragmentHTML;
+      decorateMain(adFragmentContainer);
+      await loadBlocks(adFragmentContainer);
```
Comment (kunwarsaluja) [inline scripts/delayed.js:23]: @samuelopra  can we use the fetchFragment helper here?

File: templates/universal/universal.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    width: 380px;
+    min-width: 380px;
+    z-index: 500;
+    left: 50%;
+    margin-left: -190px;
+    background-color: aqua;
+  }
+}
+
+@media (min-width: 769px) {
```
Comment (mfrisbey) [inline templates/universal/universal.css:84]: There are two media queries above for `768px` - can we combine all 3 of these into a single query? Also, does it need to be 769 and 768, or can we just use one?

**Review by mfrisbey — APPROVED** [review]
+1, I actually like the idea of having a separate include file for shared code.

I added 1 minor comment, but LGTM other than that.

**Comment by kunwarsaluja** [comment]
@mfrisbey @samuelopra Few things that might want to look at before we actually merge this
1. Apart from the homepage, the side fragment isnt loading. See example [here](https://issue-11-layout--channelco-crn-com--hlxsites.hlx.page/news/internet-of-things/5-industrial-iot-solutions-that-are-solving-big-problems)
2. Order of decoration looks weird as breadcrumb is autoblocked in eager mode, it gets loaded first and then delayed.js comes in..which created duplicated breadcrumbs within the right block
3. I think Sam is moving all contents from main in respective sections in the universal template but as breadcrumb is autoblocked as the first item under main, after the move it becomes a part of content section whereas it should be full width
4. The scroll behavior isnt implemented yet (can be done later if need be) but i think the right aside gets fixed position somewhere in the middle while scrolling

Screenshot 2023-09-22 at 3 03 49 PM

File: templates/universal/universal.css (hunk 2 above)
Comment (samuelopra) [reply templates/universal/universal.css:82]: Hey, yep the min-widths can be combined, however, I have one max-width query for the bottom nav

**Pull request merged**